CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
+  * [ExtensionManager](https://learn.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions) is moved from kube-system to internal AKS infrastructure. Clusters with extensions in canary no longer require any of the outbound networking rules documented [here](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress#cluster-extensions). Note that some extensions such as Flux still require a subset of these outbound rules.
+
+* Component Updates:
+  * Fleet networking components updated to v0.39 from v0.38 to fix CVE.
+  * [Workload Identity](https://learn.microsoft.com/azure/aks/workload-identity-deploy-cluster) updated from v1.4.0 to v1.5.0
+  * App-routing-operator has been upgraded to [v0.2.5](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#025---2025-04-22) on all supported AKS versions.
+  * Cost-analysis-agent and -scraper versions in addon charts updated from v0.0.22 to 0.0.23 to fix [CVE-2025-22871](https://pkg.go.dev/vuln/GO-2025-3563).
+  * [Cloud-controller-manager](https://cloud-provider-azure.sigs.k8s.io/blog/) updated to v1.32.4, v1.31.5, v1.30.11 and v1.29.14.
+  * [Promtheus collector](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-04-15-2025) for Azure Monitor managed service for Prometheus addon updated from 6.15.0-main-02-21-2025-4acb2b4c to 6.16.0-main-04-15-2025-d78050c6.
+  * Retina Enterprise image updated to v0.1.9 to resolve [issues](https://github.com/azure-networking/retina-enterprise/releases/tag/v0.1.9) with Retina-Operator container image build on ARM64 nodes.
+  * Retina base image to v0.0.30 for both Linux and Windows to resolve security vulnerabilities [CVE-2024-40635](https://nvd.nist.gov/vuln/detail/CVE-2024-40635) , [CVE-2025-30162](https://nvd.nist.gov/vuln/detail/CVE-2025-30162) and [CVE-2025-22870](https://nvd.nist.gov/vuln/detail/CVE-2025-22870).
+  * Calico CNI/Tigera Operator updated to [v3.28.4](https://github.com/projectcalico/calico/releases/tag/v3.28.4) (operator v1.34.10) and [v3.29.3](https://github.com/projectcalico/calico/releases/tag/v3.29.3) (operator v1.36.7), fixing [CVE-2025-0395](https://nvd.nist.gov/vuln/detail/CVE-2025-0395), [CVE-2025-22869](https://nvd.nist.gov/vuln/detail/CVE-2025-22869), [CVE-2025-22872](https://nvd.nist.gov/vuln/detail/CVE-2025-22872), [CVE-2024-9042](https://nvd.nist.gov/vuln/detail/CVE-2024-9042), [CVE-2025-0426](https://nvd.nist.gov/vuln/detail/CVE-2025-0426) and [CVE-2025-22871](https://nvd.nist.gov/vuln/detail/CVE-2025-22871).
+  * Cilium image updated to v1.13 to fix [CVE-2024-45336](https://avd.aquasec.com/nvd/cve-2024-45336), [CVE-2024-45341](https://avd.aquasec.com/nvd/cve-2024-45341), [CVE-2025-22866](https://avd.aquasec.com/nvd/cve-2025-22866), [CVE-2025-22870](https://avd.aquasec.com/nvd/cve-2025-22870), [CVE-2024-45337]( https://avd.aquasec.com/nvd/cve-2024-45337).
```

Comment:
which patch version of v1.13? and on which AKS versions? - @charleswool, @chasewilson can you folks follow up on this?